contracts/impls/trustable/WitnetRequestBoardTrustableBase.sol

```diff
@@ -674,20 +755,22 @@ abstract contract WitnetRequestBoardTrustableBase
     // ================================================================================================================
     // --- Internal functions -----------------------------------------------------------------------------------------
 
-    function _reportResult(
+    function __reportResult(
```

Comment:
Why are we adding an extra `_`?

Reply:
i'd rather prefer to prefix with two "__" any method that modifies storage. in a similar way, variables pointing to storage are also prefixed with dobule "__". this way it's much easier to follow source code and be aware when and how storage is either read or written, and therefore it eases the task of optimizing storage access, specially when dealing with loops or more complicated algorithms